config/SOUE01/rels/d_a_obj_bladeNP/symbols.txt

```diff
@@ -2,18 +2,18 @@ _prolog = .text:0x00000000; // type:function size:0x2C scope:global
 _epilog = .text:0x00000030; // type:function size:0x2C scope:global
 _unresolved = .text:0x00000060; // type:function size:0x4 scope:global
 dAcOblade_c_classInit__Fv = .text:0x00000070; // type:function size:0x90
-fn_382_100 = .text:0x00000100; // type:function size:0x98
-fn_382_1A0 = .text:0x000001A0; // type:function size:0xC4
-fn_382_270 = .text:0x00000270; // type:function size:0x8
-fn_382_280 = .text:0x00000280; // type:function size:0x30
-fn_382_2B0 = .text:0x000002B0; // type:function size:0x28
+createHeap__11dAcOblade_cFv = .text:0x00000100; // type:function size:0x98
+create__11dAcOblade_cFv = .text:0x000001A0; // type:function size:0xC4
+doDelete__11dAcOblade_cFv = .text:0x00000270; // type:function size:0x8
+actorExecute__11dAcOblade_cFv = .text:0x00000280; // type:function size:0x30
+draw__11dAcOblade_cFv = .text:0x000002B0; // type:function size:0x28
 __dt__11dAcOblade_cFv = .text:0x000002E0; // type:function size:0x68
 _ctors = .ctors:0x00000000; // type:label scope:global
 _dtors = .dtors:0x00000000; // type:label scope:global
 lbl_382_rodata_0 = .rodata:0x00000000; // type:object size:0x4 data:float
 lbl_382_rodata_4 = .rodata:0x00000004; // type:object size:0x4 align:4 data:float
```

Comment:
These two are missing `scope:local`.

`lbl_382_data_50` is actually the vtable, which is not local, so you could assign the correct vtable symbol and remove the scope:local, but that's not necessary - personally I don't care about REL symbols.txt being accurate, I'd just like there to be no linker warnings, so that part is fine too.